charts/verdaccio/values.yaml

```diff
@@ -119,7 +119,8 @@ configMap: |
 
   auth:
     htpasswd:
-      file: /verdaccio/storage/htpasswd
+      # Do not change this path if secrets htpasswd is used.
+      file: /verdaccio/auth/htpasswd
```

Comment:
`auth`?

Reply:
I thought, It makes sense to use `auth` for authentication and `storage` only for package storage. 
Besides, it would conflict with `storage` PVC.